### ISSUE TITLE
Clearly define what an Experience is

### ISSUE THREAD

**Issue by trieloff** [adobe/xdm#171]
We need a clear and unambiguous definition of Experience that links to other, defined concepts like channel, location, event, content, etc.

**Comment by trieloff** [comment]
@cdegroot-adobe

**Comment by lrosenthol** [comment]
and that is *NOT* specific to XC - but encompasses DC and CC as well...

**Comment by cmathis** [comment]
Are we waiting for @cdegroot-adobe to start off the conversation here? Is this related to the existing experienceevent schema?

**Comment by cdegroot-adobe** [comment]
@trieloff @lrosenthol  @cmathis  @kstreeter 
To be clear "experience" in this context is some sort of content or coordinated group of content (page, image, video, voice etc) that is managed and is sent to an enduser at a specific point of time based on a set of decisions made by one or more engagement applications. A marketer will have a name and maybe anid to identify the content in targeting rules and reports. 

Examples:
A user lands on the web home page of a brand. The Adobe Analytics all important pageName="homepage". In a channel agnostic world the experience.name="Homepage".
A user views the product details of a product in a mobile catalog app. In the Mobile Analytics world this is a state="product details:{sku}".   In a channel agnostic world the experience.name="Product Details".
A user selects to listen to a store opening hours in an IVR over the phone. In the IVR state change this is a state="storeOpeningHours".   In a channel agnostic world the experience.name="Store Opening Hours".

It is important to have these identification details in a channel neutral location as it is hard to ferret out the channel specific paths when we are doing omni channel queries.

**Comment by trieloff** [comment]
@cmathis yes, this came up in the experience event discussion: https://github.com/adobe/xdm/pull/143#issuecomment-371454272

**Comment by trieloff** [comment]
Here is how I see it:

- we have places where content can go. These are what we call *channel-locations* (#170). You can think of them as rectangles on a screen or in a time line (e.g. in video or audio channels)
- the stuff that is going into these places, or is contained by them, is *content*
- a large part of the discipline of marketing is to decide which *content* goes into what *channel-location*
- you can make that assignment based on the consumer's *audience* (an audience is simply a set of consumers, however defined or identified) – this process is called *targeting*
- assigning *content* to *channel-locations* across a range of *channel-locations* that span multiple impressions, interactions, channels, or even days, so that a consistent whole is created is the goal of many marketers, designers, and authors.
- creating this consistent whole of *content* in *channel-locations* is what constitutes *experience design* or *experience management*

This gets us quite close to the definition of *experience*, but we need to consider some additional points.

- consistency of the experience is in the eye of the consumer, so experiences need to be specific to an *audience* (this audience can be as broad as "everybody" or as narrow as "only the user with the ID 007")
- an experience is always concrete, so in the case of an A/B/N test, where multiple candidate variants of content are competing for the placement in a specific channel-location, every experience will have only one piece of content in each channel-location
- if there are multiple A/B/N tests, in multiple channel-locations, this will increase the potential number of possible experiences

In summary:
- an experience
  - is assigned to an audience
  - contains one or more assignments
- an assignment
  - relates to one channel-location (1:n)
  - relates to one piece of content (1:n)
  - relates to one set of eligibility rules (time, frequency, etc) (1:n)

**Comment by lrosenthol** [comment]
The problem with those definitions are that they *only* apply to DMa...But consider that we have an application on the DMe called XD (eXperience Designer) that has *nothing* to do with anything you've written here about channels or locations or ...

Especially because we have chosen to name XDM as the "Experience Data Model" (vs. having the Extensible Data Model", which is a better fit with XMP!), we need a definition of experience that applies across Adobe.

So let's try again with a wider definition please.

**Comment by cdegroot-adobe** [comment]
@lrosenthol - Is eXperience Designer not building experiences that will be shown to actual people (an audience)? 

Yes we are looking at the mechanics of consumption here, but it is not unrelated. Creators create experiences in CC, in DC and DMa we have a mix of creation and distribution and consumption analysis. 

Who from CC should we bring into this discussion to fill out the panel?

**Comment by lrosenthol** [comment]
@cdegroot-adobe Yes, in the wider scope, XD certainly builds "experiences" for customers.  However, those experiences have no channels or channel locations, no assignments, etc.   All those terms and concepts have no place in that world.

My issue is in our definition of what XDM stands for.  If we change that from eXperience->eXtensible, then the problem goes away.  If we insist on having it be eXperience - then the definition of eXperience needs to be VERY broad...

**Comment by trieloff** [comment]
@lrosenthol 

> Yes, in the wider scope, XD certainly builds "experiences" for customers. However, those experiences have no channels or channel locations, no assignments, etc. All those terms and concepts have no place in that world.

I beg to differ. The terms might not be clearly defined, but the concepts are definitely there:
> [Create a low-fidelity design in Adobe XD to define the overall structure of your app or website.](https://helpx.adobe.com/xd/how-to/web-mobile-app-ui-ux-design-basics.html?playlist=/ccx/v1/collection/product/xd/segment/designer/explevel/beginner/applaunch/ccl-get-started-1/collection.ccx.js?ref=helpx.adobe.com)

* "app or website" refers to the channel that your Adobe XD prototype is targeting. And like a channel in marketing, it comes with constraints like screen size, interaction mode, etc.

> [Define the user flow of your app or website using the prototyping tools available within Adobe XD.](https://helpx.adobe.com/xd/how-to/preview-test-prototypes-desktop-mobile.html)

* "the user flow" are the various screens or stages or artboards that the prototype contains. In the terms used above, they are the channel locations (as the most general term) with example content.

**Comment by lrosenthol** [comment]
That may be true, @trieloff - but as you yourself point out, the terms are *completely different*.  We *CANNOT* use XC-only terms here.  It MUST be a set of terms that apply equally across all three Adobe clouds - the Adobe Cloud Platform.

**Comment by trieloff** [comment]
I’d like to get someone from Adobe XD to define what an experience is. I expect that the definition will be about as wide and all-encompassing as the definition we would get out of AEM, so the question is if we can find a definition that all parties can agree upon and that is useful. If not, we will end up with an experience data model that models everything about experiences, but (because of the lack of a reliable definition) does not model experiences themselves.

**Comment by lrosenthol** [comment]
Which is **EXACTLY** why I wish we could change the X in XDM away from experience!

**Comment by bdelacretaz** [comment]
I think the word "Experience" is overloaded, we probably need to qualify it for our different use cases:

* When assembling content for a website or mobile page we are creating a _dynamic data-driven user experience_.
* When someone uses XD they are designing a _user interactions experience_, or a family of those that will be materialized using data-driven assembly as defined above.

These are the Experience Cloud and Creative Cloud variants of "Experience" that I see. If there are Document Cloud equivalent use cases they might lead to other qualifiers. 

My point is that just "Experience" is too broad and vague to address these different things.
